### PR TITLE
Adding Ledger Support to Sign TX

### LINKED ISSUE #50: Support signing with Ledger directly
It would be nice if the user could choose between signing with Keplr and signing with Ledger (via WebUSB). This has been developed and is tested in other places. It's just a matter of integrating it in the app.

Why is this relevant?
- choice
- more flexible use of different bowsers or user profiles
- easier for chains that are not yet integrated in Keplr

### PR DESCRIPTION
**Pull request by msteiner96**
Resolves #50

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,10 +11,12 @@
     "@cosmjs/amino": "^0.27.0",
     "@cosmjs/crypto": "^0.27.0",
     "@cosmjs/encoding": "^0.27.0",
+    "@cosmjs/ledger-amino": "^0.28.0",
```
[the remark this answers is not in the document]
Comment (msteiner96) [reply package.json:14]: Master is already on 0.28.0, I just merged it.

File: components/forms/TransactionSigning.js [hunk 2]
```diff
@@ -27,58 +29,73 @@ const TransactionSigning = (props) => {
...
+      (signature) => signature.signature === bases64EncodedSignature,
+    );
+
+    if (prevSigMatch > -1) {
+      setSigError("This account has already signed.");
+    } else {
+      const signature = {
+        bodyBytes: bases64EncodedBodyBytes,
+        signature: bases64EncodedSignature,
+        address: walletAccount.bech32Address,
```
Comment (webmaster128) [inline components/forms/TransactionSigning.js:94]: This line only works for the Keplr case (fixed in https://github.com/cosmos/cosmos-multisig-ui/pull/58)

File: package.json [hunk 3]
```diff
@@ -15,6 +15,7 @@
     "@cosmjs/proto-signing": "^0.28.0",
     "@cosmjs/stargate": "^0.28.0",
     "@keplr-wallet/types": "^0.9.0-alpha.4",
+    "@ledgerhq/hw-transport-webusb": "^6.24.1",
```
Comment (webmaster128) [inline package.json:18]: Also we need @cosmjs/ledger-amino, which we use directly (fixed in https://github.com/cosmos/cosmos-multisig-ui/pull/58)

**Review by webmaster128 — COMMENTED** [review]
Cool stuff, thank you! Will merge this work as part of #58

**Pull request closed without merging**